test/helper/remove_duplicates_test.go

```diff
@@ -0,0 +1,105 @@
+package helper_test
```

Comment:
I'd move this to the "test" folder, not to the "helper"

Reply:
moved it elsewhere 🤞